docs/en/deploy/Running-Alluxio-Using-Vagrant.md

```diff
@@ -162,13 +247,216 @@ and find some files written into your S3 bucket by the above tests.
 
 From a node in the cluster, you can ssh to other nodes in the cluster without password with:
 
-{% include Running-Alluxio-on-EC2/ssh-other-node.md %}
+```bash
+$ ssh AlluxioWorker1
+```
 
-## Destroy the cluster
+### Destroy the cluster
```

Comment:
Are all the "destroy the cluster" the same across all sections? Then, we should pull it out to a top level section, so it is not repeated 3 times.

Reply:
there are parts that are the same, but all three sections contain instructions specific to that deployment scenario as well.